lib/gems/pending/appliance_console/cli.rb

```diff
@@ -79,6 +79,14 @@ def time_zone?
       options[:timezone]
     end
 
+    def date_time?
+      options[:date] && options[:time]
+    end
+
+    def time_sync?
+      options[:timesync]
```

Comment:
This doesn't really make sense.

I would interpret this method's use to be "Do I need to set the time sync status". When really it's saying "Should I enable automatic time sync".

I think the issue is that this feels like there is no way to express "I want to set the automatic time sync to off"

Really, because you always need a value for this when setting the date and time (and because the option defaults to `false`) this should probably just be removed.